components/expenses/PayoutBankInformationForm.js

```diff
@@ -0,0 +1,243 @@
+
+  const renderField = (field, i) => {
+    const hasMultipleInputs = field.group.length > 1;
+    const required = field.group.some(f => f.required);
+    if (!required) {
+      return null;
+    }
+
+    const renderInput = input => {
+      const fieldName =
+        input.key === 'accountHolderName'
+          ? getFieldName(`data.${input.key}`)
+          : getFieldName(`data.details.${input.key}`);
+
+      switch (input.type) {
+        case 'text':
+          return (
+            <FastField key={input.key} name={fieldName}>
+              {({ field }) => (
+                <StyledInputField label={input.name}>
+                  {() => <Field as={StyledInput} placeholder={input.example} {...field} disabled={disabled} />}
+                </StyledInputField>
+              )}
+            </FastField>
+          );
+        case 'radio':
+        case 'select':
+          return (
+            <FastField key={input.key} name={fieldName}>
+              {({ field }) => (
+                <StyledInputField label={input.name}>
+                  {() => {
+                    const options = formatTransferWiseSelectOptions(input.valuesAllowed);
+                    return (
+                      <StyledSelect
+                        name={field.name}
+                        options={options}
+                        disabled={disabled}
+                        onChange={({ value }) => formik.setFieldValue(field.name, value)}
+                        value={options.find(c => c.value === get(formik.values, field.name))}
+                      />
+                    );
+                  }}
+                </StyledInputField>
+              )}
+            </FastField>
+          );
+        default:
+          return null;
+      }
+    };
+
+    return (
+      <Box key={i} mt={2} flex="1">
+        {hasMultipleInputs && <P fontSize="LeadParagraph">{field.name}</P>}
+        {field.group.filter(f => f.required).map(renderInput)}
+      </Box>
+    );
+  };
+
+  return (
+    <FormFieldsContainer flexDirection="column">
+      <Box mt={2} flex="1">
+        <P fontSize="LeadParagraph" fontWeight="bold">
+          {title}
+        </P>
+      </Box>
+      {Boolean(fields.length) && renderField(accountHolderFieldOptions)}
+      {fields.map(renderField)}
+    </FormFieldsContainer>
+  );
+};
+
+RequiredFields.propTypes = {
+  disabled: PropTypes.bool,
+  collective: PropTypes.shape({
+    slug: PropTypes.string.isRequired,
+  }).isRequired,
+  currency: PropTypes.string.isRequired,
+  formik: PropTypes.object.isRequired,
+  getFieldName: PropTypes.func.isRequired,
+};
+
+/**
+ * Form for a single attachment. Must be used with react-hook-form.
+ */
+const PayoutBankInformationForm = ({ isNew, getFieldName, collective }) => {
+  const formik = useFormikContext();
+  const { formatMessage } = useIntl();
+  const currencies = formatStringOptions(collective.host?.transferwise?.availableCurrencies);
```

Comment:
Nice I didn't know that optional chaining landed in our Babel yet!

Reply:
https://github.com/zeit/next.js/blob/canary/packages/next/build/babel/preset.ts#L179